lib/testobject.js

```diff
@@ -167,4 +174,32 @@ TestObject.disableTestObject = async function ({wdOverride, appiumS3Object, wd})
   await S3.deleteZip(appiumS3Object.Key);
 };
 
+TestObject.fetchAppium = async function (driverName, driverUrl) {
+  // Clone appium
+  const tempdir = await tempDir.openDir();
+  const pathToAppium = path.resolve(tempdir, 'appium');
+  await fs.rimraf(pathToAppium);
+  logger.debug('Cloning appium');
+  await exec('git', ['clone', 'git@github.com:appium/appium.git'], {cwd: tempdir});
```

Comment:
do we want to use ssh rather than http? ssh means we have to have a github-enabled ssh key on CI, right? since we're not going to push i don't see any downside to using http